doc/whats_new.rst

```diff
@@ -23,8 +23,8 @@ Legend for changelogs
 - |API|: you will need to change your code to have the same effect in the
   future; or a feature will be removed in the future.
 
+.. include:: whats_new/v0.22.rst
 .. include:: whats_new/v0.21.rst
-.. include:: whats_new/v0.20.rst
```

Comment:
It seems that we should just move the line in Previous Releases but not completely remove it.

Reply:
Very good point